api/src/main/java/org/eclipse/microprofile/metrics/Timer.java

```diff
@@ -55,13 +55,13 @@
         @Override
         void close();
     }
+
     /**
      * Adds a recorded duration.
      *
-     * @param duration the length of the duration
-     * @param unit     the scale unit of {@code duration}
+     * @param duration the length of the {@link java.time.Duration duration}
      */
-    void update(long duration, TimeUnit unit);
+    void update(Duration duration);
```

Comment:
Changing the method arguments is breaking and should definitely be in the changelog